src/index.js

```diff
@@ -13,9 +13,9 @@ export default options => {
 
   /**
    * Get rooms socket is subscribed to.
-   *
-   * @param {String} id
-   * @param {Function} [fn]
+   * Used for get rooms
+   * @param {String} id - spark's ID
+   * @param {Function} [fn] - callback
```

Comment:
I think we no longer want the callback to be optional. Also should we switch to promises?